docs/ecosystem/duckdb.md

```diff
@@ -54,50 +54,23 @@ m
     information, so the user must ensure that data has been reprojected to
     EPSG:4326.
 
-## Custom Connection
-
-If you're using a custom DuckDB connection (not the global default connection), ensure that you pass the connection object into the `con` named parameter of `viz` or `from_duckdb`:
-
-```py
-import duckdb
-from lonboard import viz
-
-con = duckdb.connect()
-sql = "SELECT * FROM spatial_table;"
-query = con.sql(sql)
-viz(query, con=con)
-```
-
 You can also render an entire table by using the `table()` method:
 
 ```py
 import duckdb
 from lonboard import viz
 
-con = duckdb.connect()
-con.execute("CREATE TABLE spatial_table AS ...;")
-viz(con.table(), con=con)
+duckdb.sql("CREATE TABLE spatial_table AS ...;")
+viz(duckdb.table("spatial_table"), con=con)
```

Comment:
There's still a `con=con` here. Is that supposed to be taken out?

Reply:
Ah oops yes!